forest/src/cli/genesis.rs

```diff
@@ -80,7 +80,9 @@ where
     } else {
         debug!("Initialize ChainSyncer with new genesis from config");
         chain_store.set_genesis(genesis_block.clone())?;
-        chain_store.set_heaviest_tipset(Arc::new(Tipset::new(vec![genesis_block.clone()])?))?;
+        futures::executor::block_on(
```

Comment:
I wonder if it would be a better to use block_on from async_std instead of the futures library

Reply:
hmmm I thought the idea of futures is that they are analogous to the executor that they are running on?  I can for sure change this though